pkg/matcher/README.md

```diff
@@ -20,20 +35,43 @@ Short Syntax
      <expr>      ::= any string
 
  Long Syntax
-
-     <line>      ::= [ <not> ] <format> <separator> <expr>
+     [ <not> ] <format> <separator> <expr>
+     
      <format>    ::= [ 'string' | 'glob' | 'regexp' | 'simple_patterns' ]
      <not>       ::= '!'
                        negative expression
      <separator> ::= ':'
      <expr>      ::= any string
 ```
 
+**Remember**: In short syntax, for example, you can enable the glob format by starting the string with a `*`, while in the long syntax you need to define it more explicitly. The following examples are identical. `simple_patterns` can be used **only** with the long syntax.
+
+Examples:
+ - Short Syntax: `'* * '`
+ - Long Syntax: `'glob':'*'`
+
 ### String matcher
-The string matcher reports whether the given value equals to the string ( use == ).
+The string matcher reports whether the given value equals to the string ( use = ).
```

Comment:
I'm not sure if the "( use = )" text is necessary here. The examples below explain the syntax, as does the table at the beginning.